docs/content/get-started/install.md

```diff
@@ -96,42 +96,17 @@ The following sections describe how to install the CLI on Linux, macOS, and Wind
 
 1. Extract the binary, `nginx-meshctl.exe`, from the zip file.
 
-1. Add the binary to your PATH and rename.
+1. Add the binary to your PATH.
 
 1. Test the installation.
 
     ```bash
-    nginx-meshctl.exe version
+    nginx-meshctl
     ```
 
-### Images
+### Air Gap Environment
```

Comment:
This is the hard part, because this `Images` header is still valid from a general perspective, and is referenced in other places in the docs. That's why we then had a nested header in this section for air gap.

Reply:
I changed the heading to Air Gap Environment because that is the only reason the user would care about images. We do not need to give the entire context and technical implementation around installation on this page. If you check many of our competitors, the installation process is completely abstracted into an install script:

https://linkerd.io/2.12/getting-started
https://istio.io/latest/docs/setup/getting-started

An install script would be awesome someday since we could eliminate all steps for all platforms and just have the script handle it. BUT I do want to ensure we still have things like image info on here, but not in detail for just "getting started".

Any other opinions from others? Happy to change anything, but do want to reduce the amount of detail we have here.

Reply:
In that case, there may be other docs that reference this header link that you'll need to update. And ensure that the context of those docs linking here still makes sense.